docs/content/how-to/system-tables.md

```diff
@@ -65,13 +65,13 @@ You can query the historical schemas of the table through schemas table.
 SELECT * FROM MyTable$schemas;
 
 /*
-+-----------+--------------------------------+----------------+--------------+---------+---------+
-| schema_id |                         fields | partition_keys | primary_keys | options | comment |
-+-----------+--------------------------------+----------------+--------------+---------+---------+
-|         0 | [{"id":0,"name":"word","typ... |             [] |     ["word"] |      {} |         |
-|         1 | [{"id":0,"name":"word","typ... |             [] |     ["word"] |      {} |         |
-|         2 | [{"id":0,"name":"word","typ... |             [] |     ["word"] |      {} |         |
-+-----------+--------------------------------+----------------+--------------+---------+---------+
++-----------+--------------------------------+----------------+--------------+---------+---------+-------------------------+
+| schema_id |                         fields | partition_keys | primary_keys | options | comment |      effective_time     |
```

Comment:
Why using `effective_time` instead of update_time?

Reply:
Upon reflection, "update_time" does indeed sound better.